instrumentation/github.com/aws/aws-sdk-go-v2/otelaws/attributes.go

```diff
@@ -38,9 +37,14 @@ var servicemap = map[string]AttributeSetter{
 	sqs.ServiceID:      SQSAttributeSetter,
 }
 
+// SystemAttr return the RPC system attribute.
+func SystemAttr(operation string) attribute.KeyValue {
+	return semconv.RPCSystemKey.String(operation)
```

Comment:
Is there ever a case where this shouldn't just use `"aws-api"` as the system?

Reply:
It seems to be the only value for aws sdk instrumentations according to [semantic convention](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/semantic_conventions/instrumentation/aws-sdk.md#common-attributes)

Reply:
Then let's have this just return a constant and not accept agreements, right?